lisa/src/main/java/it/unive/lisa/cfg/CFGDescriptor.java

```diff
@@ -39,18 +43,23 @@
 	/**
 	 * The names of the arguments of the CFG associated with this descriptor.
 	 */
-	private final String[] argNames;
+	private final Variable[] argNames;
```

Comment:
These should be called arguments since we are no longer tracking just their names